inc/zoo/swar/SWAR.h

```diff
@@ -323,6 +323,11 @@ struct BooleanSWAR: SWAR<NBits, T> {
     // Booleanness is stored in the MSBs
     static constexpr auto MaskLaneMSB =
         broadcast<NBits, T>(SWAR<NBits, T>(T(1) << (NBits -1)));
+    static constexpr auto MaskLaneLSB =
+         broadcast<NBits, T>(SWAR<NBits, T>(T(1)));
+     // Turns off LSB of each lane
+     static constexpr auto MaskLaneLSBInverse = 
+         ~broadcast<NBits, T>(SWAR<NBits, T>(T(1)));
```

Comment:
We already make `Ones`.
The name "MaskLaneLSB" is sort of ambiguous, since the SWAR of all lanes 1 can be used for arithmetic and for masking.
Let's adopt the convention that we will call masks things that naturally will turn more than one bit per lane, otherwise, let's call the constant by the name of the bit (MSB) for example, or, now that we provide bare-bones support for signed arithmetic, perhaps "Sign", unless the bit is '1', which the name Ones seems to suffice

Reply:
I followed the convention of the line previous.  Would you prefer AllLSBOn/AllMSBOn?
MaskLaneLSBInverse at S8_16 creates 0xFEFE, right?  1*0 for each lane. That name seems ok.

Reply:
The problem with `AllLSBOn` is the ambiguity of the particle "all".  In 4-bit lanes, does that mean [0x7, 0x7,...] or [0x1, 0x1,...] the "all" may refer to "all lanes" or "all non-msb bits", symmetrical objection to AllMSBOn.
The name `MaskLaneLSBInverse` seems ok, yes.  But what about `MaskNonLeastSignificantBits`? it has the particle "Mask" to indicate the intention of using it for blitting as a mask; how would people read `NonLeastSignificantBits`? would they see "1*0"?

Reply:
If only we could write bit patterns in names, but * doesn't work.